### PR TITLE
Added option to copy version information in the about dialog

### PR DESCRIPTION
**Pull request by boceckts**
Added option to copy version information of JabRef in the about dialog as described in #871 .
This will help with filing issues and bug reports as the version of JabRef, OS and Java will be copied. 

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [x] Screenshots added (for bigger UI changes)

screen shot 2016-04-25 at 22 38 58

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/*  Copyright (C) 2003-2016 JabRef contributors.
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java]: Only year 2016 as this class has been created this year, hasn't it?

File: src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    private final String OS_VERSION = System.getProperty("os.version", "unknown").toLowerCase();
+    private final String OS_ARCH = System.getProperty("os.arch", "unknown").toLowerCase();
+    private final String JAVA_VERSION = System.getProperty("java.version", "unknown").toLowerCase();
+
+	@Override
+	public void actionPerformed(ActionEvent e) {
+        String info = String.format("JabRef %s%n%s %s %s %nJava %s", Globals.BUILD_INFO.getVersion(), OS, OS_VERSION,
+                OS_ARCH, JAVA_VERSION);
+        new ClipBoardManager().setClipboardContents(info);
+		String message = String.format("%s - %s", Localization.lang("Copied_version_to_clipboard"),
```
Comment (tschechlovdev) [inline src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java]: why the indent here?

File: src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    private final String OS_ARCH = System.getProperty("os.arch", "unknown").toLowerCase();
+    private final String JAVA_VERSION = System.getProperty("java.version", "unknown").toLowerCase();
+
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        String info = String.format("JabRef %s%n%s %s %s %nJava %s", Globals.BUILD_INFO.getVersion(), OS, OS_VERSION,
+                OS_ARCH, JAVA_VERSION);
+        new ClipBoardManager().setClipboardContents(info);
+        String message = String.format(Localization.lang("Copied_version_to_clipboard"));
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java]: Don't use underscores here and remove String.format (or is this necessary?)

**Comment by tobiasdiez** [comment]
Just one minor comment, otherwise this looks good. Please rebase.

File: src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java (hunk 3 above)
Comment (boceckts) [inline src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java]: No it's not necessary this was from a previous version and is now removed.

File: src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    private final String OS = System.getProperty("os.name", "unknown").toLowerCase();
+    private final String OS_VERSION = System.getProperty("os.version", "unknown").toLowerCase();
+    private final String OS_ARCH = System.getProperty("os.arch", "unknown").toLowerCase();
+    private final String JAVA_VERSION = System.getProperty("java.version", "unknown").toLowerCase();
+
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        String info = String.format("JabRef %s%n%s %s %s %nJava %s", Globals.BUILD_INFO.getVersion(), OS, OS_VERSION,
+                OS_ARCH, JAVA_VERSION);
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java]: Creating this info version string along with the static variables should be moved to the BuildInfo class.

Comment (boceckts) [inline src/main/java/net/sf/jabref/gui/actions/CopyVersionToClipboardAction.java]: I moved them into the BuildInfo class and made them static

**Pull request merged**